### ISSUE TITLE
model.train() and model.eval()

### ISSUE THREAD

**Issue by hosseinshn** [hosseinshn/GradNorm#1]
model.train() and .eval() should be added to the network specially when there is regularizations such as Dropout.

**Comment by hosseinshn** [comment]
.train() and .eval() are added to the demo in the new version (GradNormv9).
